command/agent/config_parse.go

```diff
@@ -112,7 +107,6 @@ func ParseConfigFile(path string) (*Config, error) {
 		{"server.plan_rejection_tracker.node_window", &c.Server.PlanRejectionTracker.NodeWindow, &c.Server.PlanRejectionTracker.NodeWindowHCL, nil},
 		{"server.retry_interval", &c.Server.RetryInterval, &c.Server.RetryIntervalHCL, nil},
 		{"server.server_join.retry_interval", &c.Server.ServerJoin.RetryInterval, &c.Server.ServerJoin.RetryIntervalHCL, nil},
-		{"consul.timeout", &c.Consul.Timeout, &c.Consul.TimeoutHCL, nil},
```

Comment:
Is this being handled somewhere else? It seems like it should be handled in https://github.com/hashicorp/nomad/pull/18997/files#diff-ae0c9f058db6b0da73449cf8e5183530091a464acb31da0690550f23c26f4f97R178 for each cluster?

Reply:
Yeah, exactly. This line only did anything at all (for HCL, not JSON) b/c we had a top-level `consul.timeout` from the `Config.Consul` field, which we've just removed. The work was actually then just thrown away when we parsed the individual clusters and overwrote the default cluster!